CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.11.5
+
+* Changed Conjur Dockerfile consume conjur-base-image contains Ruby compiled OpenSSL FIPS Compliance
```

Comment:
Just realizing we're not following the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.  What do you think about updating the changelog entry to something like:
```suggestion
### Changed
* Updated FPM and Test images to use a base image with FIPS-compliant Ruby and OpenSSL.
```